pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -430,3 +452,19 @@ func (m *manager) GetAllocatableMemory() []state.Block {
 func (m *manager) GetMemory(podUID, containerName string) []state.Block {
 	return m.state.GetMemoryBlocks(podUID, containerName)
 }
+
+func (m *manager) updateAdmittedPods(pod *v1.Pod) {
```

Comment:
minor: should we try to generalize this logic?

Reply:
We can try to generalize it, but it is a small chunk of code so for now, I do not want to overcomplicate it with additional structures and interfaces.
We have some additional places that we can generalize and it is worth creating a separate PR for all generalization changes.

Reply:
fine for me.